templates/jar-runner/Dockerfile

```diff
@@ -18,7 +18,7 @@ COPY resources/ /
 
 ENTRYPOINT ["tini", "-v", "--", "/usr/local/bin/liferay_jar_runner_entrypoint.sh"]
 
-ENV JAVA_VERSION=zulu11
+ENV JAVA_VERSION=zulu21
```

Comment:
Could we switch this to keep java11 as default?  But if customers want to switch to java21 they can just put
`JAVA_VERSION=zulu21` in their dockerfile